Widgets_CDAT/textStyleEditorWidget/TextStyleEditor.py

```diff
@@ -0,0 +1,253 @@
+import vcs
+import text_style_preview
+from PySide import QtCore, QtGui
+
+
+class TextStyleEditor(QtGui.QWidget):
+
+    savePressed = QtCore.Signal(str)
+    
+    def __init__(self):
+        super(TextStyleEditor, self).__init__()
+        self.textObject = None
+
+        wrap = QtGui.QVBoxLayout()
+
+        # Set up vertical align
+        self.VAGroup = QtGui.QButtonGroup()
+
+        T_button = QtGui.QPushButton()
+        T_button.setText("Top")
+        T_button.setCheckable(True)
+        M_button = QtGui.QPushButton()
+        M_button.setText("Mid")
+        M_button.setCheckable(True)
+        B_button = QtGui.QPushButton()
+        B_button.setText("Bot")
+        B_button.setCheckable(True)
+
+        VALayout = QtGui.QHBoxLayout()
```

Comment:
Avoid using acronyms/abbreviations when naming variables and attributes; just makes people have to stare at it longer later.